ui/src/app/store/pod/types.ts

```diff
@@ -2,6 +2,12 @@ import type { TSFixMe } from "app/base/types";
 import type { Model } from "app/store/types/model";
 import type { GenericState } from "app/store/types/state";
 
+export enum PodType {
+  LXD = "lxd",
+  RSD = "rsd", // unsupported from MAAS 2.10 onward
```

Comment:
Is there any need to define this type? My understand was that it was removed on the backend as well.

Reply:
Yeah I guess it's not necessary. I added it in there just to be explicit that it's not supported, but this is probably not the right place to do that